spotbugs-tests/src/test/java/edu/umd/cs/findbugs/detect/Issue2436Test.java

```diff
@@ -0,0 +1,20 @@
+package edu.umd.cs.findbugs.detect;
+
+import edu.umd.cs.findbugs.AbstractIntegrationTest;
+import edu.umd.cs.findbugs.test.matcher.BugInstanceMatcher;
+import edu.umd.cs.findbugs.test.matcher.BugInstanceMatcherBuilder;
+import org.junit.Test;
+
+import static edu.umd.cs.findbugs.test.CountMatcher.containsExactly;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+public class Issue2436Test extends AbstractIntegrationTest {
+
+    @Test
+    public void test() {
+        performAnalysis("lambdas/Issue2436.class");
+        BugInstanceMatcher bugMatcher = new BugInstanceMatcherBuilder().build();
```

Comment:
Can you please add at least a bugtype to the BugInstanceMatcher?

Reply:
thanks, added bugtype and confidence